README.md

```diff
@@ -16,14 +16,10 @@ $ npm install -g @webosose/ares-cli
 
 ## Compatibility
 
-Our release cycle is independent of webOS OSE and Auto. We follow semver and here is the compatibility table:
+Our release cycle is independent of webOS OSE and Auto. 
+We recommend using the latest CLI. The latest CLI is compatible with the latest webOS OSE and webOS Auto"
```

Comment:
Please update "the latest webOS OSE and Auto", cause the upper sentence is 
"Our release cycle is independent of webOS OSE and Auto."

Reply:
Minor changes needed.
Code 👎

Reply:
Done